WMF Framework/WMFContentGroup+Extensions.m

```diff
@@ -543,11 +543,6 @@ - (void)updateVisibilityForUserIsLoggedIn:(BOOL)isLoggedIn {
         return;
     }
     
-#if WMF_ANNOUNCEMENT_DATE_IGNORE
```

Comment:
I did do one small project cleanup - I didn't see this used in any configurations and I would rather test as close to the Prod code as possible, so I decided to remove this macro.